projects/charger/test/test_begin_sequence.c

```diff
@@ -0,0 +1,121 @@
+#include "begin_sequence.h"
+#include "can_transmit.h"
+#include "charger_events.h"
+#include "event_queue.h"
+#include "gpio.h"
+#include "ms_test_helpers.h"
+#include "test_helpers.h"
```

Comment:
Forgot to import `unity.h`?

Reply:
Unity is included in "test_helpers.h"? I don't think there's styleguide on including transient includes, could go pretty deep with that...

Reply:
My own habit ([which is backed up by the Google C++ style guide that our style guide is based on](https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes)) is to include all the headers that define symbols used in the module (down to a reasonable level). That way e.g. if we somehow remove the `unity.h` dependency from `test_helpers.h` we won't have to add a whole bunch of #include statements all over the place. Up to you for this case, maybe it should be added to the style guide?

Reply:
Added `#include "unity.h"` to test files